app/scripts/modules/core/pipeline/config/stages/executionWindows/ExecutionWindowDayPickerComponent.js

```diff
@@ -0,0 +1,89 @@
+import angular from 'angular';
+
+class ExecutionWindowDayPickerController {
+
+  constructor() {
+    this.DAYS = [{
+      dow: 'sunday',
+      abbr: 'Sun'
```

Comment:
To be consistent with the rest of the codebase, these should probably be `key` and `label`